cmake/cmake_celix/BundlePackaging.cmake

```diff
@@ -845,9 +851,12 @@ function(install_celix_bundle_targets)
         set(EXPORT_FILE ${EXPORT_NAME}BundleTargets)
     endif ()
 
-#    if (NOT CMAKE_BUILD_TYPE STREQUAL "Release")
+    set(BASE_EXPORT_FILE ${EXPORT_FILE})
+    if (CMAKE_BUILD_TYPE)
         set(EXPORT_FILE ${EXPORT_FILE}-${CMAKE_BUILD_TYPE})
-#    endif ()
+    else ()
+        set(EXPORT_FILE ${EXPORT_FILE}-noconfig)
```

Comment:
Isn't it cleaner to use the trick that is done further down? If no build type, set build type to "NOCONFIG" and use that everywhere? Limits the build_type check to only one place.